app/retail/templates/emails/tribe_hackathon_prizes.html

```diff
@@ -31,21 +31,9 @@
     margin-top: 30px;
     background-color: #000000;
   }
-  .left {
-    text-align: left;
-  }
```

Comment:
^ I have a hunch there are other emails which also use this prop
Could we remove those as well ?

Reply:
searching the entire project for `.left` only returns `template.html`

<img width="352" alt="Screenshot 2020-10-20 at 12 20 18 PM" src="https://user-images.githubusercontent.com/36922376/96550561-a83c0200-12ce-11eb-8806-763f6a738879.png">